### PR TITLE
fix: fix content

### PR DESCRIPTION
**Pull request by Calvin-Aziszam-S**
fix code for contents

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,9 +12,16 @@
...
-					<div class="content-section">Content</div>
-					<div class="feature-a">Content for Feature A</div>
-					<div class="feature-b">Content for Feature B</div>
+					<div class="content-section">
+						<div class="feature-a">
+							<h1>Feature A</h1>
+							<div>Content for Feature A</div>
+						</div>
+						<div class="feature-b">
+							<h1>Feature B</h1>
```
Comment (prakas26) [inline index.html]: fix <h1 style="text-align: center;">

**Review by prakas26 — COMMENTED** [review]
<h1 style="text-align: center;">

File: index.html (hunk 1 above)
Comment (Calvin-Aziszam-S) [reply index.html]: noted

File: index.html [hunk 2]
```diff
@@ -12,9 +12,16 @@
...
-					<div class="content-section">Content</div>
-					<div class="feature-a">Content for Feature A</div>
-					<div class="feature-b">Content for Feature B</div>
+					<div class="content-section">
+						<div class="feature-a">
+							<h1 style="text-align: center">Feature A</h1>
+							<div>Content for Feature A</div>
+						</div>
+						<div class="feature-b">
+							<h1 style="text-align: center">Feature B</h1>
```
Comment (Calvin-Aziszam-S) [inline index.html:21]: inline style added for h1

**Pull request merged**